pkg/exporter/sumologicexporter/README.md

```diff
@@ -1,6 +1,11 @@
 # Sumo Logic Exporter
 
-**Stability level**: Beta
+**Stability level**: Deprecated
```

Comment:
We should set this in the component itself too. I think we forgot to do this for the extension as well.

Reply:
updated